Tests/ImagineEngineTests/LabelTests.swift

```diff
@@ -122,4 +122,32 @@ final class LabelTests: XCTestCase {
         XCTAssertEqual(labelBClickCount, 1)
         XCTAssertEqual(clickedLabels, [labelB, labelA, labelA])
     }
+
+    func testScaling() {
+        let label = Label(text: "Hello world")
+        let originalSize = label.size
+        let originalFontSize = label.layer.fontSize
+
+        let upscaleFactor: Metric = 2.5
+        let downscaleFactor: Metric = 0.4
+
+        // Upscale the label
+        label.scale = upscaleFactor
+
+        XCTAssertEqual(label.layer.fontSize, originalFontSize * upscaleFactor)
+        XCTAssertGreaterThan(label.size.height, originalSize.height)
```

Comment:
If you use `scale` instead of the layer, then you can simply assert that the correct scale was assigned (since we an trust Core Animation to compute the correct size).

Reply:
You're right. No need to test what's already working.

Reply:
Fixed